website/server/extractDocs.js

```diff
@@ -331,12 +333,25 @@ function parseAPIJsDocFormat(filepath, fileContent) {
   return json;
 }
 
+function debugDeepEqual(object1, object2) {
```

Comment:
Oops yeah good catch! - this is left over from the intermediate phase where I ran both doc-generators and logged all the differences. Removed it